xml/System.IO/FileInfo.xml

```diff
@@ -2137,7 +2137,7 @@ The following example demonstrates moving a file to a different location and ren
           <format type="text/markdown"><![CDATA[
 
 ## Remarks
- The <xref:System.IO.FileInfo.OpenWrite%2A> method opens a file if one already exists for the file path, or creates a new file if one does not exist. For an existing file, it does not append the new text to the existing text. Instead, it overwrites the existing characters with the new characters. If you overwrite a longer string (such as "This is a test of the OpenWrite method") with a shorter string (like "Second run"), the file will contain a mix of the strings ("Second runtest of the OpenWrite method").
+ The <xref:System.IO.FileInfo.OpenWrite%2A> method opens a file if one already exists for the file path, or creates a new file if one does not exist. For an existing file, it does not append the new text to the existing text. Instead, it overwrites the existing characters with the new characters. If you overwrite a longer string (such as "This is a test of the OpenWrite method") with a shorter string (like "Second run"), the file will contain a mix of the strings ("Second run.test the OpenWrite method.").
```

Comment:
```suggestion
 The <xref:System.IO.FileInfo.OpenWrite%2A> method opens a file if one already exists for the file path, or creates a new file if one does not exist. For an existing file, it does not append the new text to the existing text. Instead, it overwrites the existing characters with the new characters. If you overwrite a longer string (such as "This is a test of the OpenWrite method") with a shorter string (like "Second run"), the file will contain a mix of the strings ("Second run test of the OpenWrite method.").
```

Reply:
The suggestion is invalid. To tell the long story short we open a file and write:

"This is to test the OpenWrite method."

and then overwrite it with:

""Second run."

which gives us

"Second run.test the OpenWrite method."

It's not intuitive, but this is the content of the file after described change.

Reply:
Are you referring to an actual example, or just the text from the remarks? Because both of your sentences don't seem to match the remarks sentences (the first one has the word "of" and the second one doesn't have a period at the end).

Reply:
@gewarren  You are right! Thank you for your thoroughness and apologies for the confusion.